Documentation/bpf.rst

```diff
@@ -3005,12 +3010,74 @@ advantages:
   the BPF verifier that ensures the stability of the kernel's operation.
 * XDP trivially allows for atomically swapping programs during runtime without
   any network traffic interruption or even kernel / system reboot.
+* XDP allows for flexible structuring of workloads since integrated into
+  the kernel, for example, XDP can operate in busy polling or interrupt
+  driven mode, explicitly dedicating CPUs to XDP is not required, there
+  are no special hardware requirements and it does not rely on hugepages.
 * XDP does not require any third party kernel modules or licensing. It is
-  a core part of the Linux kernel and developed by the kernel community.
+  a long term architectural solution, core part of the Linux kernel and
+  developed by the kernel community.
 * XDP is already enabled and shipped everywhere with major distributions
   running a kernel equivalent to 4.8 or higher and supports most major 10G
   or higher networking drivers.
 
+XDP as a framework for running BPF in the driver addtionally ensures that
```

Comment:
addtionally --> additionally
As a framework for running BPF in the driver, XDP additionally ensures...